scripts/setup/setup-certbot

```diff
@@ -4,7 +4,8 @@ set -e
 
 usage() {
     cat <<EOF >&2
-Usage: $0 --hostname=zulip.example.com --email=admin@example.com [--method={webroot|standalone}] [--no-zulip-conf]
+Usage: $0 --email=admin@example.com [--method={webroot|standalone}] \
+[--no-zulip-conf] hostname.com another.hostname.com
```

Comment:
Perhaps `[another.hostname.com ...]` to document these extra hostname arguments as optional and repeatable.